core/src/table/table.module.css

```diff
@@ -51,3 +51,17 @@ in native HTML */
 .container tr:last-child td {
 	border-bottom-style: none;
 }
+
+/* SIZE */
+
+.small tr {
+	height: 24px;
+}
+
+.medium tr {
+	height: 32px;
+}
+
+.large tr {
+	height: 40px;
+}
```

Comment:
Currently, the spacing inside table's cells (td and th) is set at the cell's level:

https://github.com/moaijs/moai/blob/39ee98c7afb8b0fbc30f0fb003afe018b89414d2/core/src/table/table.module.css#L16-L31

This means to have the "size" prop, we should not add the "height" on top of the "padding" (which breaks the logic) but instead re-set those paddings based on user's preference.

Code-wise, this means the above code should be replaced with 3 of the referenced snippets. E.g. one of them would be:

```
/* Cell spacing */

.medium th,
.medium td {
	padding: 12px 8px;
}

.medium td:first-child,
.medium th:first-child {
	padding-left: 16px;
}

.medium td:last-child,
.medium th:last-child {
	padding-right: 16px;
}
```

Reply:
@ZeroX-DG @tuhuynh27 here you go gentlemen

Actually the Button is quite complicated than this case, since it needs several classes (to apply on several elements/cases) in each size. In the Table, we only need one class for each size so it's can be as simple as

```
interface TableSize {
  cell: string;
}

Table.sizes = {
  small: { cell: sizeStyles.small } as TableSize
  ...
}
```

Reply:
Updated, please check @thien-do, I'm still not sure about what am I doing 🤔